### ISSUE TITLE
Erroneous handling of tables with FKs to other databases

### ISSUE THREAD

**Issue by voidcastr** [phpmyadmin/phpmyadmin#11568]
Expected:
"Child" tables having FKs to "parent" tables in other databases can be browsed and searched without hassle. In the "Relation View" of such a child table, there should be three "Foreign key constraint" dropdowns next to each other, containing "parentDatabase", "parent" and "id", respectively. (see below for identifiers)

Observed:
"Child" tables having FKs to "parent" tables in other databases can be browsed and searched but error stacktraces show up at the bottom of the main div (see below). In the "Relation View" of such a child table, there are only two "Foreign key constraint" dropdowns next to each other, containing "childDatabase", and "parentDatabase", respectively, which makes no sense. (see below for identifiers)

Reproduction:
To reproduce the problem, execute the following statements:

```
CREATE DATABASE parentDatabase COLLATE utf8_general_ci;

CREATE DATABASE childDatabase COLLATE utf8_general_ci;

CREATE TABLE `parentDatabase`.`parent` (
 `id` int(11) NOT NULL AUTO_INCREMENT,
 PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

CREATE TABLE `childDatabase`.`child` (
 `id` int(11) NOT NULL AUTO_INCREMENT,
 `parentId` int(11) NOT NULL,
 PRIMARY KEY (`id`),
 KEY `parentId` (`parentId`),
 CONSTRAINT `child_ibfk_1` FOREIGN KEY (`parentId`) REFERENCES `parentDatabase`.`parent` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

----

Trying to bring up the "Search" interface now results in the following error being shown at the bottom of the main div:
```
Notice in ./libraries/relation.lib.php#1740
Undefined offset: 0

Backtrace

./libraries/relation.lib.php#1312: PMA_searchColumnInForeigners(
array,
string 'parentId',
)
./libraries/controllers/TableSearchController.class.php#933: PMA_getForeignData(
array,
string 'parentId',
boolean false,
string '',
string '',
)
./templates/table/search/rows_normal.phtml#23: PMA\Controllers\Table\TableSearchController->getColumnProperties(
integer 1,
integer 1,
)
./libraries/Template.class.php#80: include(./templates/table/search/rows_normal.phtml)
./templates/table/search/fields_table.phtml#21: PMA\Template->render(array)
./libraries/Template.class.php#80: include(./templates/table/search/fields_table.phtml)
./templates/table/search/selection_form.phtml#44: PMA\Template->render(array)
./libraries/Template.class.php#80: include(./templates/table/search/selection_form.phtml)
./libraries/controllers/TableSearchController.class.php#605: PMA\Template->render(array)
./libraries/controllers/TableSearchController.class.php#260: PMA\Controllers\Table\TableSearchController->displaySelectionFormAction()
./tbl_select.php#40: PMA\Controllers\Table\TableSearchController->indexAction()
```
For this to happen, no records need to be present.

----

After inserting records to both the parent and child table, the "Browse" view breaks as well for the child table.

```
INSERT INTO `parentDatabase`.`parent` () VALUES ();
INSERT INTO `childDatabase`.`child` (parentId) VALUES (1);
```

The following error is shown in the main div:
```
Notice in ./libraries/DisplayResults.class.php#4770
Undefined offset: 0

Backtrace

./libraries/DisplayResults.class.php#4380: PMA_DisplayResults->_setParamForLinkForeignKeyRelatedTables(array)
./libraries/sql.lib.php#1585: PMA_DisplayResults->getTable(
object,
array,
array,
)
./libraries/sql.lib.php#1863: PMA_getHtmlForSqlQueryResultsTable(
object,
string './themes/pmahomme/img/',
NULL,
array,
boolean true,
string '2',
integer 2,
NULL,
object,
array,
)
./libraries/sql.lib.php#2068: PMA_getQueryResponseForResultsReturned(
object,
array,
string 'childDatabase',
string 'child',
NULL,
NULL,
object,
string './themes/pmahomme/img/',
string '2',
integer 2,
NULL,
NULL,
NULL,
NULL,
NULL,
string 'SELECT * FROM `child`',
NULL,
)
./libraries/sql.lib.php#1946: PMA_executeQueryAndGetQueryResponse(
array,
boolean true,
string 'childDatabase',
string 'child',
NULL,
NULL,
NULL,
NULL,
NULL,
NULL,
string '',
string './themes/pmahomme/img/',
NULL,
NULL,
NULL,
string 'SELECT * FROM `child`',
NULL,
NULL,
)
./sql.php#217: PMA_executeQueryAndSendQueryResponse(
array,
boolean true,
string 'childDatabase',
string 'child',
NULL,
NULL,
NULL,
NULL,
NULL,
NULL,
string '',
string './themes/pmahomme/img/',
NULL,
NULL,
NULL,
string 'SELECT * FROM `child`',
NULL,
NULL,
)
```

----

Additional notes:
- This problem did not exist in the PMA 4.1.x series.
- Considering what the child table's "Relation View" displays, the issue's cause seems to be some namespacing/prefixing breakage.

----

Server and PMA Versions:
- Database Server
 - Server version: 5.6.15-log - MySQL Community Server (GPL)
- Web Server
 - Apache/2.4.7 (Ubuntu)
 - Database client version: libmysql - mysqlnd 5.0.11-dev - 20120503 - $Id: bf9ad53b11c9a57efdb1057292d73b928b8c5c77 $
 - PHP extension: mysqli
 - PHP version: 5.5.9-1ubuntu4.13
- phpMyAdmin 4.5.0.2

**Comment by voidcastr** [comment]
The issues also does not exist in PMA 4.4.15.

**Comment by madhuracj** [comment]
Duplicate of #11526 which is already fixed and will be part of 4.5.1 release.
